### PR TITLE
update user manual

### PR DESCRIPTION
**Pull request by elisadcg**

### CODE REVIEW

File: docs/user_manual.rst [hunk 1]
```diff
@@ -177,8 +177,35 @@ Performing (as an artist player)
 *******************************************************
 A performance involves several types of media uploads: avatars, props, and backgrounds. You chose these when you set the stage, and during a live performance they interact with each other in view of the audience. The players each control an avatar that may interact with other avatars, props, and the audience in front of a background. If the stage's creator chose music or sound effects, those media files will play.
 
+Settings of the Stage
+------------------------
+introduction bla bla bla
```
Comment (paul-rohrlach) [inline docs/user_manual.rst:182]: Can we either finish this sentence with something (instead of blah blah blah) or remove it for now?

**Review by paul-rohrlach — CHANGES_REQUESTED** [review]

**Comment by CBeez23** [comment]
I will write an introduction ASAP and remove the "blah blah blah". I'm going to go over the whole document today proofreading and adding necessary sections.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by vickis** [comment]
Good work Team

We refer to backgrounds as backdrops (its a theatre thing:)
Also the audio needs to be activated to play (so the sentence could say ..:
can rather than will be played

Don’t forget there is also text and drawing and the chat window are also
all part of a performance :)

**Comment by helenvarley** [comment]
yes great work! i'm sorry i haven't been able to contribute recently - with the workshops & travelling it's all been full-on, & now i have a week of extreme busy-ness with my other big project. next week i should have more time ... ("should" ... :laughing:

**Comment by paul-rohrlach** [comment]
Do we have any progress on this @elisadcg and @CBeez23 ?

**Comment by helenvarley** [comment]
elisa has given this link:  https://github.com/upstage-org/mobilise/blob/elisadcg-patch-12/docs/user_manual.rst
& my mission for this afternoon is to start looking at it & working on the english ... :)

**Comment by CBeez23** [comment]
I can help this evening. I've been meaning to. I'm sorry I didn't get to it before.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by helenvarley** [comment]
great, thanks @CBeez23 :smiley: i got as far as the Media section yesterday, & will carry on when i have time over the weekend. 
please check things against https://dev-app1.upstage.org.nz/V4.0/ui/ - as this is where the latest changes are. there are still some things in the rst files that are from the previous version so need to be updated, e.g. some of the terminology & workflows have been improved.

**Pull request closed without merging**